index.html

```diff
@@ -276,7 +278,49 @@ <h5>FABulator</h5>
                 href="https://github.com/FPGA-Research/FABulator?tab=readme-ov-file#getting-started"
                     target="_blank">Installation</a> </li>
 
-
+        <h3>The following tools are only needed for the CIRCT lab, but not for the
+                        hackathon. They therefore have no
+            dependencies with the previously mentioned tools.</h3>
+            <h5>Chisel</h5>
+            <ul style="padding-left: 20px;">
+              <li>
+                First install <a href="https://www.chisel-lang.org/docs/installation"
+                        target="_blank">Chisel</a> itself.
+              </li>
```

Comment:
```suggestion
              Install <a href="https://www.chisel-lang.org/docs/installation"
                        target="_blank">Chisel</a> itself.
```
I think this way it makes it a bit clearer that the given commands are just an excerpt of the install instruction.